recipe/meta.yaml

```diff
@@ -39,7 +40,6 @@ requirements:
     - python
     - pandas >=1.5.0
     - numpy >=1.20.3
```

Comment:
Since this was May 2021, and is no longer supported, I don't see any reason to allow older versions. I don't remember a specific reason for the restriction though.

Reply:
Ok no strong feelings either way. Please feel free to keep or drop as you see fit 🙂